discovery_engine_core/web-api/openapi.yaml

```
@@ -127,6 +127,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/IngestionError'
+    delete:
+      tags:
+        - documents
+      summary: Deletes all listed documents.
+      description: |-
+        Deletes all documents listed in the request body. The endpoint is
```

🎨  to be consistent with the other summaries ands descriptions
```suggestion
      summary: Delete all listed documents.
      description: |-
        Delete all documents listed in the request body. The endpoint is
```